content/swift/concepts/sets/sets.md

```diff
@@ -82,7 +82,7 @@ print("There are \(band.count) players in the band.")
 | `.remove()`              | Remove an element at a specified index                                                             |
 | `.removeAll()`           | Remove every single value from a set                                                               |
 | `.contains()`            | Check whether an item exists within the set                                                        |
-| `.intersection()`        | Populates a new set of elements with the overlapping elements of two sets                          |
+| `.intersection()`        | Returns a new set of elements with the overlapping elements of two sets                            |
```

Comment:
Could we actually remove this table? I created issues for the additional methods listed in this table.

Instead, let's keep the header and introduce the methods with a sentence, followed by `:` a colon to lead into the table that will populate as per the terms folder.

Reply:
To be clear, should it be like this, similar to the arrays?

> ### Methods
> 
> There are many set manipulation methods available in the Swift Standard Library, including generic Collection based methods. Some of these include testing the contents of sets (e.g. `contains()` and `.isEmpty`) while others can modify or manipulate the set entirely (e.g., `map()`, `reduce()`, and `sort()`). There is a list of methods applicable to the pair of sets (e.g., `.intersection()` or `.subtracting()`)
> 
> Below are some methods available for sets: